src/System.Private.ServiceModel/pkg/System.Private.ServiceModel.pkgproj

```diff
@@ -2,15 +2,11 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
-  <PropertyGroup>
-    <PreventImplementationReference>true</PreventImplementationReference>
```

Comment:
Yes, the intent is to ensure that consumers are referencing the general package itself and not an explicit OS implementation.